src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/validation/FormAValidator.java

```diff
@@ -55,8 +60,7 @@
 
     private static final String HWF_NUMBER_6_DIGITS_REGEX = "\\d{6}";
     private static final String DIVORCE_CASE_NUMBER_REGEX = "^([A-Z|a-z][A-Z|a-z])\\d{2}[D|d]\\d{5}$";
-
-    private static final String EMPTY_STRING = "";
+    private static final String GENDER_REGEX = "^(male|female|notGiven)$";
```

Comment:
should gender be check by `ALLOWED_VALUES_PER_FIELD` rather than regex?

Reply:
Yeah, that's true